### DISCUSSION TITLE
Add support for Discord bot

### DISCUSSION THREAD

**Discussion by SvetZitrka** [oobabooga/text-generation-webui#1504] [Ideas]
I would like to use this script in the "oobabooga" environment to help link language models to the Discord bot. But I can't get it to work in the "oobabooga" environment. I probably can't install what Conda module Discord :(. 

Is it possible to include this Discord bot feature in the "oobabooga" environment or reveal how to install the Discord module?

**Comment by bmoconno** [comment]
You already have another post #1503 about this, please don't spam the boards. There's already too much here to filter through.

**Comment by matatonic** [comment]
Once the openai extension is merged, the official discord bot at https://github.com/openai/gpt-discord-bot could be usable (maybe with some changes), I haven't tested this yet but is on my list of things to get working.

**Reply by matatonic** [reply]
This is done, see https://github.com/oobabooga/text-generation-webui/issues/1503

**Comment by matatonic** [comment]
The openai API extension was merged, there are a number of options for discord bots that work with the openai api.

**Comment by chrisrude** [comment]
I also threw this together.  Similar to a few other efforts, but I put a little more polish in.  The openai API might be nice too, though sometimes there are benefits when the API can be changed by the community.

https://github.com/chrisrude/oobabot
